lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -823,6 +825,14 @@ def _download_direct_download_link(self, file_url, copy_file_path):
             with closing(online_file):
                 with open(copy_file_path, 'wb') as output:
                     shutil.copyfileobj(online_file, output)
+            # check first 5 lines of file content
+            with open(copy_file_path) as copied_file:
+                for line in islice(copied_file, 5):
+                    if line.lower().find('html') != -1:
+                        raise ValueError("Undownable File.\n" + 
+                            "Please make sure file is publicly accessible\n" +
+                            "File URL: %s" % file_url)
```

Comment:
I guess first I need to understand what this code is for. Why do you want to exclude html files from download? It seems unlikely for our use case but it's possible someone might want to download an html file.